src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.test.tsx

```diff
@@ -46,83 +46,64 @@ const mockedActivity = {
   isPerformanceTask: false,
   hasAnswer: true,
 };
-
-jest.mock('react-hook-form', () => ({
-  ...jest.requireActual('react-hook-form'),
-  useWatch: jest.fn(),
-  useFormContext: () => ({
-    getValues: () => ({
-      startDate: new Date('2024-01-17T15:27:08.736Z'),
-      endDate: new Date('2024-01-23T15:27:08.736Z'),
-      startTime: '00:00',
-      endTime: '23:59',
-      filterByIdentifier: false,
-      identifier: [],
-      versions: [
-        {
-          id: '2.0.0',
-          label: '2.0.0',
+const mockedAnswers = [
+  {
+    answerId: 'answer-id',
+    decryptedAnswer: [
+      {
+        activityItem: {
+          question: {
+            en: 'Single Selected - Mocked Item',
+          },
+          responseType: 'singleSelect',
+          responseValues: {
+            options: [
+              {
+                id: '484596cc-0b4e-42a9-ab9d-20d4dae97d58',
+                text: '1',
+                isHidden: false,
+                value: 0,
+              },
+              {
+                id: 'a6ee9b74-e1d3-47b2-8c7f-fa9a22313b19',
+                text: '2',
+                isHidden: false,
+                value: 1,
+              },
+            ],
+          },
+          config: {
+            removeBackButton: false,
+            skippableItem: true,
+            randomizeOptions: false,
+            timer: 0,
+            addScores: false,
+            setAlerts: false,
+            addTooltip: false,
+            setPalette: false,
+            additionalResponseOption: {
+              textInputOption: false,
+              textInputRequired: false,
+            },
+          },
+          name: 'ss-1',
+          isHidden: false,
+          allowEdit: true,
+          id: 'ab383cc6-834b-45da-a0e1-fc21ca74b316',
+          order: 1,
         },
-        {
-          id: '2.0.1',
-          label: '2.0.1',
+        answer: {
+          value: '0',
+          edited: null,
         },
-      ],
-    }),
-  }),
-}));
-
-jest.mock('modules/Dashboard/hooks', () => ({
-  ...jest.requireActual('modules/Dashboard/hooks'),
-  useDecryptedActivityData: jest.fn(),
-}));
-
-jest.mock('./ReportFilters', () => ({
-  ReportFilters: () => <div data-testid="report-filters"></div>,
-}));
-
-jest.mock('./ActivityCompleted', () => ({
-  ActivityCompleted: () => <div data-testid="report-activity-completed"></div>,
-}));
-
-jest.mock('./Subscales', () => ({
-  Subscales: () => <div data-testid="report-subscales"></div>,
-}));
-
-jest.mock('./ResponseOptions', () => ({
-  ResponseOptions: () => <div data-testid="report-response-options"></div>,
-}));
-
-jest.mock('downloadjs', () => jest.fn());
-
-describe('Report component', () => {
-  test('renders Report correctly with data', async () => {
-    mockAxios.get.mockResolvedValueOnce({
-      data: {
-        result: [
+        items: [
```

Comment:
I guess it reduces the code duplicate if you move the object from row 54 in a variable and use it here (the same) or just skip items as empty array. I will investigate in another  task if `decryptedAnswer` can be used without `items` property at all.